content/en/docs/instrumentation/net/getting-started/logs.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Logs
+description: Getting Started with OpenTelemetry .NET Logging
+weight: 2
+---
+
```

Comment:
This should have a description about OTel logs with .NET and how it's based on a core library, similar to tracing